index.html

```diff
@@ -902,111 +1346,127 @@
               </p>
               <!-- btn -->
               <a
                 href="#"
-                class="text-sm font-medium text-white py-[14px] px-5 border-button bg-for-logo-section rounded-lg"
+                class="text-sm font-medium text-white py-3.5 px-5 border border-[#262626] bg-[#1A1A1A] rounded-lg"
               >
                 Read More
               </a>
             </div>
 
             <!-- second question -->
             <div
-              class="2xl:w-[30%] xl:w-1/3 sm:w-1/2 hidden p-10 border-button rounded-xl sm:flex flex-col items-start gap-6"
+              class="sm:col-span-2 lg:col-span-2 hidden p-10 border border-[#262626] rounded-xl sm:flex flex-col justify-between items-start gap-6"
             >
               <!-- first div for rating -->
-              <p class="font-semibold text-xl text-color">
+              <p class="font-semibold text-xl text-[#FFFFFF]">
                 What documents do I need to sell my property through Estatein?
               </p>
               <!-- experience info -->
-              <p class="font-medium text-base text-color-addition-info">
+              <p class="font-medium text-base text-[#999999]">
                 Find out about the necessary documentation for listing your
                 property with us.
               </p>
               <!-- client card and his name -->
               <a
                 href="#"
-                class="text-sm font-medium text-white py-[14px] px-5 border-button bg-for-logo-section rounded-lg"
+                class="text-sm font-medium text-white py-3.5 px-5 border border-[#262626] bg-[#1A1A1A] rounded-lg"
               >
                 Read More
               </a>
             </div>
 
             <!-- third question -->
             <div
-              class="2xl:w-[30%] w-1/3 xl:flex hidden p-10 border-button rounded-xl flex-col items-start gap-6 justify-between"
+              class="lg:col-span-2 lg:flex hidden p-10 border border-[#262626] rounded-xl flex-col items-start gap-6 justify-between"
             >
               <!-- first div for rating -->
-              <p class="font-semibold text-xl text-color">
+              <p class="font-semibold text-xl text-[#FFFFFF]">
                 How can I contact an Estatein agent?
               </p>
               <!-- experience info -->
-              <p class="font-medium text-base text-color-addition-info">
+              <p class="font-medium text-base text-[#999999]">
                 Discover the different ways you can get in touch with our
                 experienced agents.
               </p>
               <!-- client card and his name -->
               <a
                 href="#"
-                class="text-sm font-medium text-white py-[14px] px-5 border-button bg-for-logo-section rounded-lg"
+                class="text-sm font-medium text-white py-3.5 px-5 border border-[#262626] bg-[#1A1A1A] rounded-lg"
```

Comment:
this border color is used at so many places, add it to tailwind config

Reply:
changed